src/bionics.cpp

```diff
@@ -776,14 +774,14 @@ void player::process_bionic( int b )
     }
 }
 
-void bionics_uninstall_failure( player *u )
+void player::bionics_uninstall_failure( std::string installer_name )
 {
     switch( rng( 1, 5 ) ) {
         case 1:
-            add_msg( m_neutral, _( "You flub the removal." ) );
+            add_msg( m_neutral, _( "%s flub the removal." ), installer_name );
```

Comment:
In case `%s` is a NPC, it will be "flubs", "messes" and "screws". Maybe change it to `add_msg_if_player_or_npc` here and below?